src/components/UI/atoms/layouts/FlexContainer/FlexContainer.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '<variables>/mainTheme';
+
+
+/**
+ * @description - Container Component
+ *
+ * @prop {node} children - children
+ * @prop {string} margin - margin
+ * @prop {string} padding - padding
+ * @prop {string} display - display
+ * @prop {string} backgroundColor - background color
+ * @prop {string} borderColor - border color
+ * @prop {string} borderRadius - add border radius
+ * @prop {string} boxShadow - boxShadow
+ *
+ * @return {component} FlexContainer
+ */
+const FlexContainer = ({
+  children,
+  margin,
+  padding,
+  display,
+  backgroundColor,
+  borderColor,
+  borderRadius,
+  boxShadow,
+}) => (
+  <FlexContainer.Container
+    margin={margin}
+    padding={padding}
+    display={display}
+    backgroundColor={backgroundColor}
+    borderColor={borderColor}
+    borderRadius={borderRadius}
+    boxShadow={boxShadow}
+  >
+    {children}
+  </FlexContainer.Container>
+);
+
+FlexContainer.Container = styled.div`
+  ${({
+    margin,
+    padding,
+    display,
+    borderRadius,
+    boxShadow,
+    backgroundColor,
+    borderColor,
+    theme: {
+      spacing, backgroundColors, borderColors, boxShadows,
+    },
+  }) => `
+    display: ${display};
+    background-color: ${backgroundColors[backgroundColor]};
+    margin: ${spacing[margin]};
+    padding: ${spacing[padding]};
+    border-radius: ${spacing[borderRadius]};
+    box-shadow: ${boxShadows[boxShadow]};
+    border: 1px solid ${borderColors[borderColor] || 'transparent'};
+  `}
+`;
+
+FlexContainer.defaultProps = {
+  display: 'flex',
+  backgroundColor: 'white',
```

Comment:
There are different degrees of white, you need to have a more expansive name for this white. 
maybe ```white001: #FFF;
             white002: #fCF2D8;```